src/cassandra.go

```diff
@@ -67,21 +70,43 @@ func main() {
 	e, err := entity(i)
 	fatalIfErr(err)
 
-	var opts []jmx.Option
-	if args.Verbose {
-		opts = append(opts, jmx.WithVerbose())
+	jmxConfig := &gojmx.JMXConfig{
+		Hostname:         args.Hostname,
+		Port:             int32(args.Port),
+		Username:         args.Username,
+		Password:         args.Password,
+		RequestTimeoutMs: int64(args.Timeout),
+		Verbose:          args.Verbose,
 	}
 
 	if args.KeyStore != "" && args.KeyStorePassword != "" && args.TrustStore != "" && args.TrustStorePassword != "" {
-		ssl := jmx.WithSSL(args.KeyStore, args.KeyStorePassword, args.TrustStore, args.TrustStorePassword)
-		opts = append(opts, ssl)
+		jmxConfig.KeyStore = args.KeyStore
+		jmxConfig.KeyStorePassword = args.KeyStorePassword
+		jmxConfig.TrustStore = args.TrustStore
+		jmxConfig.TrustStorePassword = args.TrustStorePassword
 	}
 
-	fatalIfErr(jmx.Open(args.Hostname, strconv.Itoa(args.Port), args.Username, args.Password, opts...))
-	defer jmx.Close()
+	jmxClient := gojmx.NewClient(context.Background())
+	_, err = jmxClient.Open(jmxConfig)
+	log.Debug("nrjmx version: %s", jmxClient.GetClientVersion())
```

Comment:
Can we also add a `log.Debug()` with the JMX config? I think it might be useful to ensure that all options have been picked correctly.